docker/release/nanoserver/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# escape=`
```

Comment:
@vors - I double-checked my thinking on this with @raghushantha this afternoon and he agreed that using this docker processor directive aids in making the dockerfile simpler to read\maintain.  If you look at the way Docker is executing powershell, it's doing it with a -command param, so all that follows is processed as a single lines with standard Powershell escapes.